ckanext/scheming/helpers.py

```diff
@@ -275,7 +280,7 @@ def date_tz_str_to_datetime(date_str):
         microseconds = int(m.groupdict(0).get('microseconds'))
         time_tuple = time_tuple[:5] + [seconds, microseconds]
 
-    final_date = datetime.datetime(*map(int, time_tuple))
+    final_date = datetime.datetime(*list(map(int, time_tuple)))
```

Comment:
this `list()` is unnecessary. Even better would be
```python
final_date = datetime.datetime(*(int(x) for x in time_tuple))
```
then we could remove the `import map` above